conference.js

```diff
@@ -1733,6 +1735,43 @@ export default {
             : Promise.resolve();
     },
 
+    /**
+     * Toggle noise suppression on the current audio local track. This will apply a
+     * {@code NoiseSuppressionEffect}.
+     */
+    async toggleNoiseSuppression() {
+        const state = APP.store.getState();
+        const dispatch = APP.store.dispatch;
+        const noiseSuppressionActive = isNoiseSuppressionActive(state);
+        const localAudio = getLocalJitsiAudioTrack(APP.store.getState());
+
+        if (!localAudio) {
+            logger.warn('Can not toggle noise suppression without any local track active.');
```

Comment:
Shouldn't this be a notification? Should the control be disabled so we don't end up here?

Reply:
added notifications